src/custom-types.json

```diff
@@ -583,6 +583,13 @@
       "http://facebook.github.io/react/docs/getting-started.html"
     ]
   },
+  "text/vue": {
+    "compressible": true,
+    "extensions": ["vue"],
+    "sources": [
+      "http://vuejs.org/v2/guide/single-file-components.html"
```

Comment:
Yes, it is necessary. Otherwise, you need to follow https://github.com/jshttp/mime-db#adding-custom-media-types